File: server/webapi/DAL/Personal.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace DAL
{
    public class Personal
    {
        [Key]
        [Required]
        [MinLength(9), MaxLength(9)]
        public string Tz { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
    }
}

File: client/src/app/Classes/Personal.ts
export class Personal {
    Tz:string;
    Name:string;
    Address:string;
    Phone:string;
    Email:string;
    constructor(Tz?:string,Name?:string,Address?:string,
        Phone?:string,Email?:string){
            this.Tz=Tz;
            this.Name=Name;
            this.Address=Address;
            this.Phone=Phone;
            this.Email=Email;
    }
}
